iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -265,6 +419,35 @@ public void processCustomEvent(CustomEvent event) throws IOException {
         }
     }
 
+    private Map<String, Object> attemptTypeConversion(Map<String, String> attributes) {
+        if (attributes == null) {
+            return null;
+        }
+        Map<String, Object> converted = new HashMap<>(attributes.size());
+        attributes.forEach((key,value)-> {
+            if (isEmpty(value)) {
```

Comment:
It can't be null. Very unlikely for it to be empty and if it is I don't think that's actually an issue here.